From d5455909938610519b4578e6d6d27178d90ad040 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 28 Apr 2025 10:37:55 +0200
Subject: [PATCH] dt-bindings: net: fix STM32 TSN switch bindings

Fix issues raised by dt_binding_check:
./Documentation/devicetree/bindings/net/stm32-ttt.yaml:5:6:
[error] string value is redundantly quoted with any quotes (quoted-strings)
./Documentation/devicetree/bindings/net/stm32-ttt.yaml:6:10:
[error] string value is redundantly quoted with any quotes (quoted-string

Change-Id: I3d7db9feb9a916babe5def5dabbbcef7f22a2d44
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/454451
Domain-Review: Christophe ROULLIER <christophe.roullier@foss.st.com>
Reviewed-by: Christophe ROULLIER <christophe.roullier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe ROULLIER <christophe.roullier@foss.st.com>
---
 Documentation/devicetree/bindings/net/stm32-ttt.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/net/stm32-ttt.yaml b/Documentation/devicetree/bindings/net/stm32-ttt.yaml
index db70a4a4baaa..02050c6ee76a 100644
--- a/Documentation/devicetree/bindings/net/stm32-ttt.yaml
+++ b/Documentation/devicetree/bindings/net/stm32-ttt.yaml
@@ -2,8 +2,8 @@
 # Copyright 2019 BayLibre, SAS
 %YAML 1.2
 ---
-$id: "http://devicetree.org/schemas/net/stm32-ttt.yaml#"
-$schema: "http://devicetree.org/meta-schemas/core.yaml#"
+$id: http://devicetree.org/schemas/net/stm32-ttt.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
 
 title: STMicroelectronics STM32 TSN switch glue layer controller
 
-- 
2.50.1

